server/addresses.js

```diff
@@ -41,19 +40,17 @@ router.route('/:addressType')
 router.route('/:addressType/:addressId')
 .get(mustBeLoggedIn, (req, res, next) => {
   req.targetUser['get' + req.addressType]({id: req.params.addressId})
-  .then(shippingInfos => {
-    res.json(shippingInfos)
-  })
+  .then(shippingInfos => res.json(shippingInfos))
   .catch(next)
 })
 .put(mustBeLoggedIn, (req, res, next) => {
   req.targetUser['get' + req.addressType]({id: req.params.addressId})
   .then(targetAddress => targetAddress.update(req.body))
-  .then(updatedAddress => res.json(updatedAddress))
+  .then(updatedAddress => res.json(req.targetUser))
```

Comment:
decide between res.json-ing the updated Address or the entire updated user body -- then change Promise names accordingly